web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -1841,6 +1841,15 @@
                   : "";
             return angular.isFunction(this[fnName]) ? this[fnName]() : false;
           },
+          canViewMetadataHistory: function () {
+            var profile = gnConfig["metadata.history.accesslevel"] || 'Editor',
```

Comment:
If you compile the code, you should notice that there are some formatting changes that occur.  i.e. all the ' will change to " and some indentations may change as well.  Youl should commit the newly formatted code otherwise it will affect other developper later.

Reply:
i fixed the formatting issue. It was due to copying from 3.12 source code. Now they are the same